library/components/text/text.css

```diff
@@ -0,0 +1,75 @@
+.h1,.h2,.h3,.h4,.h5,.h6{
+    margin: 0;
```

Comment:
Just a suggestion, you can include margin: 0; in reset css

Reply:
Hey Amartya, I did it intentionally for the heading component to remove the block-level and inline-level margins. 
My library will not have default reset styles (reset is just for the documentation page) , I want users to define it as per their preference.

Thanks for your suggestion 😃